lib/porky_lib.rb

```diff
@@ -1,10 +1,21 @@
 # frozen_string_literal: true
 
 module PorkyLib
+  require 'porky_lib/caching_encryptor'
+  require 'porky_lib/concerns/has_guid'
+  require 'porky_lib/concerns/has_encrypted_attributes'
+  require 'porky_lib/concerns/partition_provider'
   require 'porky_lib/config'
   require 'porky_lib/file_service_helper'
   require 'porky_lib/file_service'
+  require 'porky_lib/models/application_record'
+  require 'porky_lib/models/encryption_key'
+  require 'porky_lib/services/key_management_service'
   require 'porky_lib/symmetric'
+  require 'porky_lib/validators/string_validator'
   require 'porky_lib/version'
   require 'porky_lib/unencrypted'
+  require 'porky_lib/caching_encryptor'
+  require 'porky_lib/concerns/has_encrypted_attributes'
+  require "porky_lib/railtie" if defined?(Rails) # for the rake tasks
```

Comment:
Why double quotes here? Also for readability, can you please keep things in alphabetical order?

Reply:
Ok, fixed.